src/app/styles/components/_variables.scss

```diff
@@ -6,3 +6,10 @@ $headerbar-background: #466BC1;
 $button-hover-color: #3859A5;
 $button-padding: 12px;
 $logout-button-color: #FFFFFF;
+
+// Introduction stage variables
+$selection-yellow-background: #FBF38F;
+$blue: #04529D;
```

Comment:
you are defeating the purpose of scss variables as well
have a look at few suggested articles on how to name color variables in scss (there isn't a rule written in stone, so really it's up to the coder to decide, but please be consistent.)

here's some links with a quick googling:
https://davidwalsh.name/sass-color-variables-dont-suck
https://coryetzkorn.com/blog/color-variables-in-sass/
https://alexpate.uk/journal/naming-sass-color-variables/